package.json

```diff
@@ -49,20 +47,22 @@
     "lint-staged": "^15.2.10"
   },
   "resolutions": {
-    "**/nomnom/underscore": "^1.12.1",
-    "**/core/ember-inline-svg/**/nth-check": "^2.0.1",
-    "**/ember-cli/**/ansi-html": "^0.0.9",
-    "**/ember-cli/testem/fireworm/async": "^2.6.4",
-    "**/@hashicorp/design-system-components/ember-stargate": "^0.6.0",
-    "micromatch": "^4.0.8"
+    "nomnom/underscore": "^1.12.1",
```

Comment:
yarn 4 automatically removed all the glob resolutions but I still had to manually add these to avoid errors

Reply:
Unrelated but I gotta say I was a bit confused when I read it's only one level of depth and then see `@babel/core/@babel/generator` until I remembered packages can also have slashes 😮‍💨

Reply:
I agree with @hashicc comment. Taking `ansi-html` for example, the current resolution will force `ansi-html` specific version within the entire codebase, when the issue its only within `ember-cli`.

Reply:
thanks for the feedback and the examples, makes sense to keep it relative.